dev-docs/bidders/minutemedia.md

```diff
@@ -32,27 +32,50 @@ The MinuteMedia adapter requires setup and approval. Please reach out to hb@minu
 
 ## Example
 ```javascript
-var adUnits = [
-       {
-        code: 'dfp-video-div',
-        sizes: [[640, 480]],
-        mediaTypes: {
+var adUnits = [{
+      code: 'banner-div',
+      mediaTypes: {
+          banner: {
+              sizes: [
+                  [300, 250],
+                  [728, 90]
+              ]
+          }
+      },
+      bids: [{
+          bidder: 'minutemedia',
+          params: {
+              org: '56f91cd4d3e3660002000033', // Required
+              floorPrice: 0.05, // Optional
+              placementId: '12345678', // Optional
+              testMode: false // Optional
+          }
+      }]
+  },
+  {
+      code: 'dfp-video-div',
+      sizes: [
+          [640, 480]
+      ],
+      mediaTypes: {
           video: {
-            playerSize: [[640, 480]],
-            context: 'instream'
+              playerSize: [
+                  [640, 480]
+              ],
+              context: 'instream'
           }
-        },
-        bids: [{
+      },
+      bids: [{
           bidder: 'minutemedia',
           params: {
-            org: '56f91cd4d3e3660002000033', // Required
-            floorPrice: 5.00, // Optional
-            placementId: '12345678', // Optional
-            testMode: false // Optional
+              org: '56f91cd4d3e3660002000033', // Required
+              floorPrice: 5.00, // Optional
```

Comment:
Is this still relevant? Shouldn't the floor price module take care of it?

Reply:
the adapter preferring the  floor price module. but in case he doesn't exists , the adapter will take the value from the param